app/frontend/styles/_inset-text.scss

```diff
@@ -1,17 +1,24 @@
+.app-inset-text {
+  border-left-width: $govuk-border-width;
+}
```

Comment:
I think we should use a modifier for this, rather than just swapping out the `govuk-` prefix for `app-`. Maybe `app-inset-text--narrow`?

Reply:
Nice! Have used `.app-inset-text--narrow-border` to be more explicit about what is narrow.